src/Components/HighChartsWrapper/HighChartsWrapper.types.ts

```diff
@@ -6,10 +6,10 @@ import { IDataHistoryAggregationType } from '../../Models/Types/Generated/3DScen
 export const MAX_NUMBER_OF_SERIES_IN_HIGH_CHARTS = 10;
 
 export interface IHighChartSeriesData {
-    name: string; // name of the series which is used in legend and tooltip
+    name: string /**  name of the series which is used in legend and tooltip*/;
```

Comment:
sorry, now these are doing to label the wrong properties I think. :D the /** comments have to be on the line above I think?

Reply:
it should work now